charts/spire/charts/spire-server/templates/configmap.yaml

```diff
@@ -157,6 +157,38 @@ plugins:
   {{- end }}
   {{- end }}
 
+  {{- with .Values.upstreamAuthority.vault }}
+  {{- if eq (.enabled | toString) "true" }}
+  {{- $upstreamAuthorityUsed = add1 $upstreamAuthorityUsed }}
+  UpstreamAuthority:
+    vault:
+      plugin_data:
+        vault_addr: {{ .vaultAddr | quote }}
+        pki_mount_point: {{ .pkiMountPoint | quote }}
+        insecure_skip_verify: {{ .insecureSkipVerify }}
+        {{- if ne (.insecureSkipVerify | toString) "true" }}
+        ca_cert_path: "/run/spire/vault-upstream/tls.crt"
```

Comment:
shouldn't the ca be `ca.crt`

e.g. comparing this to secrets created via cert-manager they have following keys.

```
ca.crt
tls.crt
tls.key
```

In that case tls.crt could be e.g. the hashi vault server certificate when using cert-manager to create a certificate for hashi-vault.

Reply:
Ah I did not see this, yes I will update this to make it standard 👍